MOBILE_CLIENT/COMMON/_IMPORTANT/PACKAGE_OBFUSCTAED/description.md

```diff
@@ -0,0 +1 @@
+The application code is obfuscated which is limiting detection capabilities.
```

Comment:
We want a finding unique to Flutter. Please rename the finding and indicate everywhere this is related to Flutter.